bin/run-browser-tests-ci-batch

```diff
@@ -41,4 +41,4 @@ for item in batch_contents:
 
 shell_command = ["./bin/run-browser-tests-ci"] + batch_contents
 
-subprocess.run(shell_command)
+subprocess.run(shell_command, check=True)
```

Comment:
Not blocking: I believe this can be replaced with the equivalent 
 `subprocess.check_call(shell_command)` (https://docs.python.org/3/library/subprocess.html#subprocess.check_call).

Reply:
The docs say that run is the newer and preferred method. https://docs.python.org/3/library/subprocess.html#using-the-subprocess-module